piplicenses.py

```diff
@@ -162,6 +162,9 @@ def normalize_pkg_name(pkg_name: str) -> str:
         lambda metadata: metadata.get("maintainer-email"),
     ],
     "license": [lambda metadata: metadata.get("license")],
```

Comment:
```suggestion
        lambda metadata: metadata.get("license-expression"),
        lambda metadata: metadata.get("license")
```

isn't it just enough to add a new lambda here?

Reply:
Technically it would be. However, wouldn't be possible to tell if a package has adopted PEP 639 yet or is still using the old `License` key. That's why I've chosen to use a separate mapping here.